src/conversions.rs

```diff
@@ -183,6 +331,9 @@ pub fn mouse_button(button: MouseButton) -> iced_core::mouse::Button {
         MouseButton::Right => Button::Right,
         MouseButton::Middle => Button::Middle,
         MouseButton::Other(val) => Button::Other(val),
+        // TODO: We need https://github.com/iced-rs/iced/pull/2169
```

Comment:
this has been merged
https://github.com/iced-rs/iced/pull/2169

Reply:
fixed in [e003361](https://github.com/tasgon/bevy_iced/pull/24/commits/e003361034712798a71a5ffa0ea921dc4fe634e6)